recipes/pysoundcard/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pysoundcard" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/soundcard/SoundCard-{{ version }}.tar.gz
+  sha256: 410835514ba10809803cb9887d4270f392b59eaf365915bb94516af3f8b1d037
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pulseaudio  # [linux]
```

Comment:
If this is optional and you still get a functional Windows and macOS package without it you can use the OS-noarch: https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies